### PR TITLE
fix: css: global css variable name alignment with design specification

### PR DESCRIPTION
**Pull request by dkilgore-eightfold**
## SUMMARY:
The Figma design specification of foundational base colors uses a superset naming convention for six colors out of each base set of 10 to assist with identifying the color and its relation to the UI, in code these are to be:

--[BASECOLOR]-**background1**-color
--[BASECOLOR]-**background2**-color
--[BASECOLOR]-**background3**-color
--[BASECOLOR]-**tertiary**-color
--[BASECOLOR]-**secondary**-color
--[BASECOLOR]-color

The important part of the naming convention is in **bold** above. In code (differing from Figma) the base color will simply be named something to the effect of `red-color`, `blue-color`, etc. for a couple of reasons:

1. We wish to avoid deprecating the already existing CSS variable representing the base color of each set
2. The `-color` part keeps it in line with the others in the superset

Updates the following base color percentages to match the latest spec:

`--red-color-60`
  OLD: `#c45353`
  NEW: `#c15151`

`--disruptive-color-60`
  OLD: `#c45353`
  NEW: `#c15151`

`--orange-color-60`
  OLD: `#ce811a`
  NEW: `#c97e19`

`--yellow-color-60`
  OLD: `#ad9300`
  NEW: `#a88f00`

[FIGMA (internal to Eightfold only)](https://www.figma.com/file/SlKRC7oKF7XZyHMv2op4ch/Octuple-DS-(Theme-2)?type=design&node-id=11686-119298&t=AWSn672DWWSEGZXY-0)

## JIRA TASK (Eightfold Employees Only):
ENG-53463

## CHANGE TYPE:

- [x] Bugfix Pull Request
- [ ] Feature Pull Request

## TEST COVERAGE:

- [x] Snapshots and stories have been updated

## TEST PLAN:
Pull the PR branch and run `yarn` and `yarn storybook`. Verify all component colors are behaving as expected.

### CODE REVIEW

**Review by ypike-eightfold — APPROVED** [review]

File: src/styles/themes/_default-theme.scss [hunk 1]
```diff
@@ -6,201 +6,291 @@
...
-  --red-color-30: #ffa3a3; // $color-background3-red (Surface 3)
+  --red-color-10: #ffefef;
+  --red-color-20: #ffc6c6;
+  --red-color-30: #ffa3a3;
   --red-color-40: #f48686;
   --red-color-50: #e46f6f;
-  --red-color-60: #c45353; // $color-tertiary-red (Graphics)
-  --red-color-70: #993838; // $color-secondary-red (Text)
-  --red-color-80: #6c2222; // $color-primary-red (Extra high contrast)
+  --red-color-60: #c15151;
```
Comment (jhoward-eightfold) [inline src/styles/themes/_default-theme.scss:52]: intentional value change?

File: src/styles/themes/_default-theme.scss [hunk 2]
```diff
@@ -6,201 +6,291 @@
...
-  --disruptive-color-30: #ffa3a3; // $color-background3-disruptive (Surface 3)
+  --disruptive-color-10: #ffefef;
+  --disruptive-color-20: #ffc6c6;
+  --disruptive-color-30: #ffa3a3;
   --disruptive-color-40: #f48686;
   --disruptive-color-50: #e46f6f;
-  --disruptive-color-60: #c45353; // $color-tertiary-disruptive (Graphics)
-  --disruptive-color-70: #993838; // $color-secondary-disruptive (Text)
-  --disruptive-color-80: #6c2222; // $color-primary-disruptive (Extra high contrast)
+  --disruptive-color-60: #c15151;
```
Comment (jhoward-eightfold) [inline src/styles/themes/_default-theme.scss:73]: seems like it

File: src/styles/themes/_default-theme.scss [hunk 3]
```diff
@@ -6,201 +6,291 @@
...
-  --orange-color-30: #ffcd78; // $color-background3-orange (Surface 3)
+  --orange-color-10: #fff8eb;
+  --orange-color-20: #ffe3b0;
+  --orange-color-30: #ffcd78;
   --orange-color-40: #ffb650;
   --orange-color-50: #f29d31;
-  --orange-color-60: #ce811a; // $color-tertiary-orange (Graphics)
-  --orange-color-70: #9d6309; // $color-secondary-orange (Text)
-  --orange-color-80: #6c4500; // $color-primary-orange (Extra high contrast)
+  --orange-color-60: #c97e19;
```
Comment (jhoward-eightfold) [inline src/styles/themes/_default-theme.scss:111]: whats with the color tweaks? assuming this is per the spec

**Review by jhoward-eightfold — APPROVED** [review]
hoping you are confident here. let the record show, I did not sanity check this visually

File: src/styles/themes/_default-theme.scss (hunk 1 above)
Comment (dkilgore-eightfold) [reply src/styles/themes/_default-theme.scss:52]: Yes. Was not 100% in line with the Figma spec.

File: src/styles/themes/_default-theme.scss (hunk 2 above)
Comment (dkilgore-eightfold) [reply src/styles/themes/_default-theme.scss:73]: Yep

File: src/styles/themes/_default-theme.scss (hunk 3 above)
Comment (dkilgore-eightfold) [reply src/styles/themes/_default-theme.scss:111]: Yes

**Pull request merged**